src/main/java/org/fundacion/jala/converter/view/compiler/Console.java

```diff
@@ -12,28 +12,35 @@
 
 import javax.swing.JPanel;
 import javax.swing.JTextArea;
+import javax.swing.JScrollPane;
+import javax.swing.border.Border;
 import javax.swing.border.EmptyBorder;
-import java.awt.BorderLayout;
-import java.awt.Dimension;
+import javax.swing.border.LineBorder;
+import java.awt.Font;
 import java.awt.Color;
+import java.awt.BorderLayout;
 
 /**
  * This class customizes a panel as a console.
  */
 class Console extends JPanel {
+    private final int MARGIN = 10;
+    private final int FONT_SIZE = 12;
     private JTextArea console;
-    private final int WIDTH_C = 600;
-    private final int HEIGHT_C = 200;
 
     protected Console() {
         console = new JTextArea();
         console.setBackground(Color.black);
         console.setForeground(Color.white);
-        console.setEditable(false);
-        console.setPreferredSize(new Dimension(WIDTH_C, HEIGHT_C));
-        console.setBorder(new EmptyBorder(0, 0, 0, 0));
+        console.setEditable(true);
+        console.setBorder(new EmptyBorder(MARGIN, MARGIN, MARGIN, MARGIN));
+        console.setFont(new Font("Courier New", Font.PLAIN, FONT_SIZE));
+        Border border = new LineBorder(Color.WHITE, 0);
+        JScrollPane sp = new JScrollPane(console);
```

Comment:
The variable should have a more descriptive name.

Reply:
Solved!